addons/test_website/static/tests/tours/custom_snippets.js

```diff
@@ -23,8 +23,7 @@ const wTourUtils = require('website.tour_utils');
  */
 
 wTourUtils.registerWebsitePreviewTour('test_custom_snippet', {
-    url: '/',
-    edition: true,
+    url: '/@/?enable_editor=1',
```

Comment:
Commit message "This commit also removes the getClientActionUrl util" -> no, deprecated